kata-containers-image/debian.control

```diff
@@ -0,0 +1,13 @@
+Source: kata-containers-image
+Section: devel
+Priority: optional
+Maintainer: Kata containers team <https://github.com/kata-containers/>
+Standards-Version: 3.9.6
+Homepage: https://katacontainers.io
+Build-Depends: debhelper (>= 9), flex, bison
+Debtransform-Tar: kata-containers.tar.gz
+
+Package: kata-containers-image
+Architecture: amd64
```

Comment:
Question, how does obs handle this to build for multiples archs?

Reply:
I don't know actually, never done that before.

Reply:
@jcvenegas You can configure that on the OBS site to specify the architectures, or specify on the `osc` command line while building locally.